test/over_react/component_declaration/flux_component_test.dart

```diff
@@ -157,13 +163,52 @@ void main() {
     });
 
     test('should not redraw after being unmounted', () async {
-      var renderedInstance = render(TestDefault());
-      TestDefaultComponent component = getDartComponent(renderedInstance);
+      var renderedInstance = render(testComponents.basic());
+      dynamic component = getDartComponent(renderedInstance);
       component.componentWillUnmount();
       component.redraw();
       await nextTick();
       expect(component.numberOfRedraws, equals(0));
     });
+
+    group('calls super in the appropriate lifecycle methods', () {
+      test('componentWillMount', () {
+        expect(() {
+          mount(TestPropValidation()());
+        }, throwsPropError_Required('TestPropValidationProps.required'),
+            reason: 'should have called super, triggering prop validation logic');
+      });
+
+      test('componentWillReceiveProps', () {
+        var jacket = mount((testComponents.propValidation()..required = 'foo')());
+        expect(() {
+          jacket.rerender(testComponents.propValidation()());
+        }, throwsA(anyOf(
+          hasToStringValue('V8 Exception'), /* workaround for https://github.com/dart-lang/sdk/issues/26093 */
```

Comment:
The Dart SDK issue has been closed, and apparently should be fixed as of 1.24.1?

Reply:
Workaround removed in https://github.com/Workiva/over_react/pull/193/commits/20bbd9e83a8f6bdd7ee04fdba711a3bc4d4dc710